org.mypaint.MyPaint.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.mypaint.MyPaint",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "mypaint",
+    "rename-desktop-file": "mypaint.desktop",
+    "rename-appdata-file": "mypaint.appdata.xml",
+    "rename-icon": "mypaint",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "scons",
+            "cleanup": [ "*" ],
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/scons/scons/2.5.1/scons-2.5.1.tar.gz",
+                    "sha256": "0b25218ae7b46a967db42f2a53721645b3d42874a65f9552ad16ce26d30f51f2"
+                }
+            ],
+            "build-commands": [
+                "python setup.py install --prefix=/app"
+            ]
+        },
+        {
+            "name": "swig",
+            "config-opts": ["--without-boost", "--without-pcre"],
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz",
+                    "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+                }
+            ]
+        },
+        {
+            "name": "numpy",
+            "buildsystem": "simple",
+            "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://downloads.sourceforge.net/project/numpy/NumPy/1.11.2/numpy-1.11.2.tar.gz",
+                        "sha256": "04db2fbd64e2e7c68e740b14402b25af51418fc43a59d9e54172b38b906b0f69"
+                    }
+            ],
+            "build-commands": [
+                "python2 setup.py build -j 4",
+                "python2 setup.py install --prefix /app"
+            ]
+        },
+        {
+            "name": "pycairo",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/pygobject/pycairo/releases/download/v1.13.1/pycairo-1.13.1.tar.gz",
+                    "sha256": "d8f58de67ddd01eda9e5112de57599b7d0154d71c9474821e98866c228794641"
+                }
+            ],
+            "build-commands": [
+                "python2 setup.py build",
+                "python2 setup.py install --prefix /app"
+            ]
+        },
+        {
+            "name": "pygobject",
```

Comment:
@cosimoc the problem is not the specific version, but building for Python2. The GNOME runtime has `pygobject` and `pycairo` built for Python 3, and MyPaint has yet to be ported from Python 2 to 3.